frontend/src/app/testResults/TestResultTextModal.tsx

```diff
@@ -0,0 +1,120 @@
+import { gql, useMutation } from "@apollo/client";
+import Modal from "react-modal";
+import moment from "moment";
+
+import Button from "../commonComponents/Button/Button";
+import { showAlertNotification } from "../utils";
+import { formatFullName } from "../utils/user";
+import "./TestResultCorrectionModal.scss";
+import {
+  InjectedQueryWrapperProps,
+  QueryWrapper,
+} from "../commonComponents/QueryWrapper";
+
+export const testQuery = gql`
+  query getTestResultForText($id: ID!) {
+    testResult(id: $id) {
+      patientLink {
+        internalId
+      }
+      dateTested
+      patient {
+        firstName
+        middleName
+        lastName
+        birthDate
+        phoneNumbers {
+          type
+          number
+        }
+      }
+    }
+  }
+`;
+interface patientPhoneDetails {
```

Comment:
FYI this interface already exists as `PhoneNumber` and is available globally:

https://github.com/CDCgov/prime-simplereport/blob/00b878874b3078f42f0374970d2a6f1d7f407a7d/frontend/src/app/Settings/types.d.ts#L95-L98